ds-caselaw-ingester/lambda_function.py

```diff
@@ -211,21 +246,25 @@ def copy_file(tarfile, input_filename, output_filename, uri, s3_client: Session.
 def send_retry_message(
     original_message: Dict[str, Union[str, int]], sqs_client: Session.client
 ) -> None:
-    retry_number = int(original_message["number-of-retries"]) + 1
-    if retry_number <= int(os.getenv("MAX_RETRIES", "5")):
-        retry_message = {
-            "consignment-reference": original_message["consignment-reference"],
-            "s3-folder-url": "",
-            "consignment-type": original_message["consignment-type"],
-            "number-of-retries": retry_number,
-        }
-        sqs_client.send_message(
-            QueueUrl=os.getenv("SQS_QUEUE_URL"), MessageBody=json.dumps(retry_message)
-        )
+    if is_v1(original_message):
+        retry_number = int(original_message["number-of-retries"]) + 1
+        if retry_number <= int(os.getenv("MAX_RETRIES", "5")):
+            retry_message = {
+                "consignment-reference": original_message["consignment-reference"],
+                "s3-folder-url": "",
+                "consignment-type": original_message["consignment-type"],
+                "number-of-retries": retry_number,
+            }
+            sqs_client.send_message(
+                QueueUrl=os.getenv("SQS_QUEUE_URL"),
+                MessageBody=json.dumps(retry_message),
+            )
+        else:
+            raise MaximumRetriesExceededException(
+                f'Maximum number of retries reached for {original_message["consignment-reference"]}'
+            )
     else:
-        raise MaximumRetriesExceededException(
-            f'Maximum number of retries reached for {original_message["consignment-reference"]}'
-        )
+        raise MaximumRetriesExceededException("v2 messages cannot yet be retried")
```

Comment:
I remember when discussing this new s3 messaging set up with the other devs in TNA, the consensus was that the only reason we needed to retry messages previously was because the presigned urls would expire. Since the new system doesnt use preseigned urls anymore, we shouldn't need to retry

Reply:
That seems reasonable.